Remora.Rest/Remora.Rest.csproj

```diff
@@ -7,7 +7,7 @@
         </PackageReleaseNotes>
     </PropertyGroup>
 
-    <PropertyGroup>
+    <PropertyGroup Condition="'$(TargetFramework)' != 'netstandard2.1'">
```

Comment:
You can probably use one of the `IsXXX` properties from Remora.Sdk here to futureproof somewhat.
https://github.com/Remora/Remora.Sdk/blob/main/Remora.Sdk/Sdk/Sdk.targets#L21

Reply:
since incompatibility with NAOT is to my understanding tied to ref assembly surface, this would have to be re-evaluated for every non-`netx.x` target regardless

Reply:
Not sure I understand what you mean by that - could you clarify?

Reply:
there's nothing to future proof here, because there's no indications whether or not future netstandard versions, if any, will support NAOT; so unless you consider adding netstandard2.0 as a target there's nothing gained, nothing lost from using `IsNETStandard`. i can use these properties, it just doesn't really future-proof anything